src/ZendeskApi_v2/Requests/HelpCenter/Sections.cs

```diff
@@ -10,7 +10,7 @@ namespace ZendeskApi_v2.Requests.HelpCenter
     public interface ISections : ICore
     {
 #if SYNC
-        GroupSectionResponse GetSections();
```

Comment:
please leave the old method and add this as a new overload. adding it as default parameters makes the updated Library binary incompatible for drop in upgrades.